app/controllers/dashboard_controller.rb

```diff
@@ -448,6 +466,41 @@ def initiate_saml_login
     javascript_redirect saml_protected_page
   end
 
+  # Initiate an OpenIDC Login from the main login page
+  def initiate_oidc_login
+    javascript_redirect oidc_protected_page
+  end
+
+  # Login support for OpenIDC - GET /oidc_login
+  def oidc_login
+    request.env.each do |key, value|
+      $log.info("    request.env[#{key}] = #{value}") if key =~ /^HTTP_/
+    end
```

Comment:
probably left over from debugging all this fun stuff

Reply:
Good. Catch. 👍